README.md

```diff
@@ -89,28 +92,43 @@ On all platforms:
 One might choose the semi dockerized approach for:
 
 1. faster startup/teardown time (not having to rebuild/start/stop containers)
-1. better ide integration
+1. better IDE integration
 
 Notes:
 
 - [osx catalina, reinstall command line tools](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d)
 
-- [poetry](https://python-poetry.org/docs/#installation)
+- Install [poetry](https://python-poetry.org/docs/#installation)
 
-### Semi Dockerized Setup
+##### Semi Dockerized Setup Steps
 
-1.  Pre reqs (macOs instructions)
+1.  Pre reqs  
+    macOS instructions:
 
         brew install postgresql llvm openssl yarn
 
         echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
         export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
 
+    Ubuntu 20.04 instructions:
+        
+        # general deps (also see `poetry` link above)
+        sudo apt install postgresql llvm openssl yarn
+
+        # add'l deps* for poetry / python setup
+        sudo apt install python3.9 python3.9-dev libpq5=12.9-0ubuntu0.20.04.1
+        sudo apt install libpq-dev
+
+    _*Notes_
+    - _the specific libpq5 version shown here is required for libpq-dev at time of writing_
+    - _python3.9 and python3.9-dev are required per poetry config, but **should not** be made the system default on Ubuntu 20.04 (i.e., leave python3 and python3.8 as-is)_
```

Comment:
I'm not sure how it behaves on Ubuntu but poetry should include a venv specific python version which is used within its venv, like on my osx it looks like

```
[app-9BVIRsKI-py3.9]jaredlockhart@Jareds-MacBook-Pro~/Documents/experimenter(main)$ which python
/Users/jaredlockhart/Library/Caches/pypoetry/virtualenvs/app-9BVIRsKI-py3.9/bin/python
```

so maybe that's helpful instead of installing it at a system level and having to think about which is the default system python?

Reply:
Unless I confused myself during setup, I believe I had to install it on the system for it to be available to poetry. Looking a little more closely now, I think if I had `pyenv` installed, poetry might've been able to figure it out for me, but otherwise I needed `python3.9` on my `PATH`, which was handled via the `apt install python3.9`. 

https://python-poetry.org/docs/managing-environments/#switching-between-environments

Reply:
Oh yes you're right, I was just doing it again on another machine last night and yes you do have to use pyenv first for poetry to find the right version, so actually it's

1) pyenv to install and set 3.9 as the local python
2) poetry in that context to create the venv with that version